nav2_dwb_controller/dwb_critics/test/map_grid_test.cpp

```diff
@@ -0,0 +1,238 @@
+/*
```

Comment:
I think you're missing `scoreTrajectory`, `propogateManhattanDistances`, and `scorePose` coverage

Reply:
They will be tested in the GoalDist and PathDist tests. As I can't create useful values using the base class, I can't test them in this base test.

Reply:
Really? You can't input some basic 10x10 costmap and work with it?

Reply:
I can only only make maps with unreachable and obstacle values with this class.

Reply:
https://github.com/ros-planning/navigation2/blob/main/nav2_dwb_controller/dwb_critics/src/goal_dist.cpp

You have an instance deriving from this base class, you can give it a costmap and then call the prepare functions like this does to populate the distances / score functions. You've have to update your fixture class to do a little more